prisma/seed.ts

```diff
@@ -1,21 +1,23 @@
 import { PrismaClient } from '@prisma/client';
-import { encodePassword } from '../src/utils/bcrypt';
+import { encodePassword } from '../src/utils/encodePassword';
+import * as dotenv from 'dotenv';
```

Comment:
**suggestion**: Consider moving the dotenv configuration to the entry point of your application to ensure all modules have access to environment variables.